Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -1,15 +1,41 @@
-import os
-import sys
-import re
-import uuid
+"""
+This is the main script that runs pyabc and CellModeller. 
+
+Instructions:
+1. Set cellmodeller_module to the name of the simulation module (e.g., Tutorial1a.py)
+2. Set max_cells to the number of cells where the simulation will end
+3. Set the summary statistics to calculate in the model() function
+4. Define the ABC calibration settings in the main function
+5. Set experimental values for summary statistics in the main function;
+    ensure the chosen summary statistics are the same as the simulated ones
+6. Define prior distributions for each parameter in the main function
+7. Run by entering the command: python pyabc_cellmodeller.py
+"""
+
+# Standard python modules
 import multiprocessing
 import matplotlib.pyplot as plt
 import numpy as np
-import pyabc
+import os
+import re
+import sys
+import uuid
+
+# Installed modules
 from CellModeller.Simulator import Simulator
+import pyabc
+
+# Local/custom modules
+sys.path.append(os.path.join(os.path.expanduser('~'), 'CellModeller-ingallslab/Scripts/summaryStatistics/biophysics/'))
+import anisotropy
+import aspect_ratio
+import density_calculation
+import growth_rate_analysis
+sys.path.append(os.path.join(os.path.expanduser('~'), 'CellModeller-ingallslab/Scripts/summaryStatistics/'))
+import helperFunctions
 
-cellmodeller_module = "simulation_module.py"    
-max_cells = 100
+cellmodeller_module = "simulation_module.py"  
+max_cells = 1000
 
 def model(parameters):
```

Comment:
Please write in the comments what does the function return? (by looking at the function's name it's not clear what it returns)